### PR TITLE
Deprecate classic dashboard docs

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

On Jan 8, 2024, classic dashboards will be officially deprecated. This PR is to create the redirect URLs and delete the classic dashboard docs.

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206238446803263

### CODE REVIEW

**Comment by jpipkin1** [comment]
@JV0812, I converted this PR to a draft since the build-and-deploy check is failing. Put the PR back into review once you've got it fixed and it's ready for us to look at.

**Comment by jpipkin1** [comment]
@JV0812, you write in the PR description "This PR is to create the redirect URLs and delete the classic dashboard docs." But this PR doesn't delete the classic dashboard docs. It only removes articles from the sidebar. If you also intend for this PR to delete the classic dashboard articles, you'll have to remove all the files under https://github.com/SumoLogic/sumologic-documentation/tree/main/docs/dashboards-classic (and provide redirects for them).

**Comment by JV0812** [comment]
Thanks @jpipkin1, I have deleted the classic dashboards now, so the statement can remain the same in the PR descroption..

File: docs/manage/users-roles/roles/role-capabilities.md [hunk 1]
```diff
@@ -58,16 +58,16 @@ Following are the capabilities you can assign when you [create roles](create-ma
 | Manage Support Account Access	| Enable management of the Sumo Logic [support account](/docs/manage/security/enable-support-account) for your organization.|
 | Manage Audit Data Feed	| Enable and manage the [Audit Index](/docs/manage/security/audit-indexes/audit-index), which provides information on the internal events that occur in your account associated with account management, user activity, and scheduled searches.|
 | Manage SAML	| Provision and manage [SAML](/docs/manage/security/saml) for single sign-on to your Sumo Logic accounts.|
-| Manage Share Dashboards Outside of Organization	| [Share a dashboard](/docs/dashboards-classic/share-dashboard-outside-org) with users who don't have access to Sumo Logic.|
+| Manage Share Dashboards Outside of Organization	| Share a dashboard with users who don't have access to Sumo Logic.|
```
Comment (jpipkin1) [inline docs/manage/users-roles/roles/role-capabilities.md:61]:
```suggestion
| Manage Share Dashboards Outside of Organization	| [Share a dashboard](/docs/dashboards/share-dashboard-outside-org/) with users who don't have access to Sumo Logic.|
```

**Review by jpipkin1 — APPROVED** [review]
Wow, @JV0812! You did a wonderful job! I only found a couple of minor things. 

In addition to my suggested change above, you'll also have to update a reference to "classic dashbards" in this article:
https://help.sumologic.com/docs/search/get-started-with-search/build-search/dynamic-parsing/#use-dynamic-parsing

File: blog-service/2017/12-31.md [hunk 2]
```diff
@@ -595,7 +593,7 @@ See [Share Dashboards ](/docs/dashboards-classic/share-dashboard-inside-org) and
...
 
-**Share Dashboards Outside of Your Organization**. You can share your live dashboards in view-only mode with no sign-in required, with an option to restrict access to viewers connecting from IPs / CIDRs specified in your service whitelist. This feature must be enabled by an administrator on the **Manage > Security > Sumo Logic Policies** page. See [Share Dashboards](/docs/dashboards-classic/share-dashboard-inside-org) for more information.
+**Share Dashboards Outside of Your Organization**. You can share your live dashboards in view-only mode with no sign-in required, with an option to restrict access to viewers connecting from IPs / CIDRs specified in your service whitelist. This feature must be enabled by an administrator on the **Manage > Security > Sumo Logic Policies** page.
```
Comment (kimsauce) [inline blog-service/2017/12-31.md:596]:
```suggestion
**Share Dashboards Outside of Your Organization**. You can share your live dashboards in view-only mode with no sign-in required, with an option to restrict access to viewers connecting from IPs / CIDRs specified in your service allowlist. This feature must be enabled by an administrator on the **Manage > Security > Sumo Logic Policies** page. 
```

File: docs/contributing/glossary.md [hunk 3]
```diff
@@ -67,13 +67,13 @@ This glossary provides an alphabetized list of technical term definitions used i
...
 
 **[Data Forwarding](/docs/manage/data-forwarding)**. When enabled, the Data Forwarding feature allows Sumo Logic to upload data to an Amazon S3 bucket that belongs to your organization. Log messages are saved as CSV files in compressed gzip files. They are accumulated and returned right after being ingested by Sumo Logic.
 
-**[Data Panel](/docs/dashboards-classic/chart-panel-types)**. Panels are placed on Dashboards and provide a graphical representation of your organization's data. Data Panels are created by running search queries. From the resulting data in the Search page's Aggregates tab, you can display that data using different types of charts. Once Panels are created, they are saved to a Dashboard. See also, Text Panel.
+**[Data Panel](/docs/dashboards/panels)**. Panels are placed on Dashboards and provide a graphical representation of your organization's data. Data Panels are created by running search queries. From the resulting data in the Search page's Aggregates tab, you can display that data using different types of charts. Once Panels are created, they are saved to a Dashboard. See also, Text Panel.
```
Comment (kimsauce) [inline docs/contributing/glossary.md:76]:
```suggestion
**[Data Panel](/docs/dashboards/panels)**. Panels are placed on Dashboards and provide a graphical representation of your organization's data. Data Panels are created by running search queries. From the resulting data in the Search page's Aggregates tab, you can display that data using different types of charts. Once Panels are created, they are saved to a Dashboard. See also, Text Panels.
```

File: docs/contributing/glossary.md [hunk 4]
```diff
@@ -294,7 +294,7 @@ This glossary provides an alphabetized list of technical term definitions used i
 
 ## T
 
-**[Text Panels](/docs/dashboards-classic/get-started/add-text-panel)**. Used in Dashboards, you can add Text Panels to include titles or text descriptions. See also Markdown.
+**[Text Panels](/docs/dashboards/panels/#text-panel)**. Used in Dashboards, you can add Text Panels to include titles or text descriptions. See also Markdown.
```
Comment (kimsauce) [inline docs/contributing/glossary.md:297]:
```suggestion
**[Text Panel](/docs/dashboards/panels/#text-panel)**. Used in Dashboards, you can add Text Panels to include titles or text descriptions. See also Markdown.
```

File: docs/manage/users-roles/roles/role-capabilities.md [hunk 5]
```diff
@@ -58,16 +58,16 @@ Following are the capabilities you can assign when you [create roles](create-ma
...
 
 ## Dashboards
 
 | Capability | Description |
 | :-- | :-- |
-| Share Dashboards With The World	| [Share dashboards](/docs/dashboards-classic/share-dashboard-outside-org) in view-only mode with no login required. Anyone with the URL can view the dashboard without logging in.|
-| Share Washboards With The Allowlist	| [Share dashboards](/docs/dashboards-classic/share-dashboard-inside-org) in view-only mode with no login required. Viewers must be connecting from IP addresses specified in your [service allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses).|
+| Share Dashboards With The World	| [Share dashboards](/docs/dashboards/share-dashboard-outside-org) in view-only mode with no login required. Anyone with the URL can view the dashboard without logging in.|
+| Share Washboards With The Allowlist	| [Share dashboards](/docs/dashboards/share-dashboard-new/) in view-only mode with no login required. Viewers must be connecting from IP addresses specified in your [service allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses).|
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/role-capabilities.md:70]:
```suggestion
| Share Dashboards with the world	| [Share dashboards](/docs/dashboards/share-dashboard-outside-org) in view-only mode with no login required. Anyone with the URL can view the dashboard without logging in.|
| Share Dashboards with your allowlist	| [Share dashboards](/docs/dashboards/share-dashboard-new/) in view-only mode with no login required. Viewers must be connecting from IP addresses specified in your [service allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses).|
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
couple minor things!

**Pull request merged**